tests/unittests/test_text_classification.py

```diff
@@ -0,0 +1,31 @@
+import os
+import autogluon as ag
+from autogluon import TextClassification as task
+
+def test_fit():
+    dataset = task.Dataset(name='ToySST')
+    predictor = task.fit(dataset,
+                         net=ag.Categorical('bert_12_768_12'),
+                         pretrained_dataset=ag.Categorical('book_corpus_wiki_en_uncased'),
+                         epochs=1,
+                         num_trials=1,
+                         batch_size=4,
+                         dev_batch_size=4,
+                         max_len=16)
+    test_acc = predictor.evaluate(dataset)
+    print('finished')
+
+def test_custom_dataset_fit():
```

Comment:
CI Failed:

tests/unittests/test_text_classification.py Fatal Python error: Fatal Python error: AbortedAborted

Can this be fixed? Also, could you try to update test to work on CPU? Once it does, I can try this locally to get a better idea of how the code is structured.

Reply:
This will probably work best since my understanding of the code may be somewhat incorrect according to this test. I thought that the user would import the CustomTSVTask and create it themselves, but it appears this is not the case.

Reply:
I tried to fix the ci however it took 3h to make a pass, and though it is so strange we do not have a GPU machine to run the ci. We should have one. It can of course run on the CPU. I am not sure why you want to run the code locally in order to help you understand the code structure. If you really want to understand, you could actually leverage a GPU machine that could make the process faster.

Reply:
I do not quite understand what you mean by " the code is incorrect" according the test. You have not ran the code yet based on your previous comment. For now the user just need to provide the dataset path to enable custom dataset usage. It should be easier than what you suggested, and it is the same with the tabular dataset design.

Reply:
I did not say the code is incorrect, I said that my understanding of the how the code worked was likely not accurate. Regarding running its locally, it is so I can use the python debugger from my Mac instead of debugging from a remote machine which is more difficult.

I believe that the CI machine does has a GPU, but you can ask @zhanghang1989 if this is the case.

Reply:
Actually you can leverage python pdb to debug on remote machines: https://docs.python.org/3/library/pdb.html. It could save time in lots of cases, e.g., especially when we are dealing with deep and large models. 

From the ci information, I think we have one gpu node "
NODE_NAME=linux-gpu-g4dn-2x-6", however it could be not enough to test some large models when there is a need with reasonable time limits.